common/version.go

```diff
@@ -51,7 +51,9 @@ func (v Version) IsCompatible(verRcv Version) bool {
 		return true
 	case v.Major == 1 && verRcv.Major == 1 && verRcv.Minor >= 4:
 		return true
-	case v.Major == 2 && verRcv.Major == 1 && verRcv.Minor >= 5:
+	case v.Major == 1 && v.Minor >= 5 && verRcv.Major == 2 && verRcv.Minor == 0:
```

Comment:
can probably remove some of the 1.4 cases above now

Reply:
True, especially since we're actually not compatible with 1.4 anymore